orders.js

```diff
@@ -1,246 +1,296 @@
+const series = require("run-series");
+const n = require("numbro");
 
-var series = require("run-series");
-var n = require('numbro');
+const dates = require("./isodate.js");
+const instruments = require("./instruments.js");
+const account = require("./account.js");
+const positions = require("./positions.js");
+const conf = require("./conf.js");
+const quotes = require("./quotes.js");
 
-var dates = require('./isodate.js');
-var instruments = require('./instruments.js');
-var account = require('./account.js');
-var positions = require('./positions.js');
-var conf = require('./conf.js');
-var quotes = require('./quotes.js');
+const sell_orders = [];
```

Comment:
sell_orders, buy_orders, stop_orders and backtest_orders are not constants, they must be var